app/assets/javascripts/app/views/help_view.js

```diff
@@ -187,8 +197,23 @@ app.views.Help = app.views.StaticContentView.extend({
     e.preventDefault();
   },
 
+  chat: function(e){
+    this.renderStaticSection("chat", "faq_chat", this.CHAT_SUBS);
+    this.menuClicked(e);
+
+    e.preventDefault();
+  },
+
   linkHtml: function(url, text) {
     return "<a href=\"" + url + "\" target=\"_blank\">" + text + "</a>";
+  },
+
+  linkImage: function(src, alt){
+    return '<div class="help-image"><img src="images/' + src + '" alt="' + alt + '"/></div>';
```

Comment:
Is there something special to do for an image to be taken in account ?
Because :
![injection1](https://cloud.githubusercontent.com/assets/3429231/6316627/fc34c194-ba2d-11e4-87fb-1bcc909b49c7.png)
and though :
![injection2](https://cloud.githubusercontent.com/assets/3429231/6316628/03bdc24e-ba2e-11e4-8032-bae0c42cacc6.png)